### DISCUSSION TITLE
I would like to enable custom domains for my project

### DISCUSSION THREAD

**Discussion by dhruvbhatia7** [vercel/community#353] [Help] [answered]
I'm currently using wildcard subdomains for customers which is working great. Examples:
```
site1.myapp.com
site2.myapp.com
```

Now I want to enable custom domains for my customers, for example:
`feedback.customerwebsite.com -> site1.myapp.com` (for customer with site1.myapp.com subdomain on my platform)

So how do I map custom subdomains to wildcard subdomains?

**Comment by samcx** [comment, marked as answer]
There are probably multiple ways to do this, but you will need some sort of rewrites strategy → https://vercel.com/docs/cli#project-configuration/rewrites.

Let us know if you have any further questions or concerns.
